### PR TITLE
Step2

### PR DESCRIPTION
**Pull request by ndraper2**

### CODE REVIEW

File: templates/base.jinja2 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <h1>My Python Journal</h1>
+      <section id="content">
+      {% block body %}{% endblock %}
+      </section>
+    </main>
+    <footer>
+      <p>Created in the Code Fellows Python Dev Accelerator</p>
+    </footer>
+  </body>
+</html>
```
Comment (cewing) [inline templates/base.jinja2:28]: Add a newline here.  It's even good style in an html file.

File: test_journal.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+from sqlalchemy.exc import IntegrityError
+
+TEST_DATABASE_URL = os.environ.get(
+    'DATABASE_URL',
+    'postgresql://ndraper2@localhost:5432/test-learning-journal'
+)
+os.environ['DATABASE_URL'] = TEST_DATABASE_URL
+os.environ['TESTING'] = "True"
+
+import journal
```
Comment (cewing) [inline test_journal.py:15]: Keep all imports grouped together.  It will save you confusion in the future looking for them.

File: test_journal.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+
+def test_entry_no_text(db_session):
+    bad_data = {'title': 'test_title'}
+    journal.Entry.write(session=db_session, **bad_data)
+    with pytest.raises(IntegrityError):
+        db_session.flush()
+
+
+def test_entry_extra_field(db_session):
```
Comment (cewing) [inline test_journal.py:81]: I like this one.  Are you handling `TypeError` in your add view?

File: test_journal.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    bad_data = {'title': 'test title', 'text': 'test text', 'footer': 'test footer'}
+    with pytest.raises(TypeError):
+        journal.Entry.write(session=db_session, **bad_data)
+        db_session.flush()
```
Comment (cewing) [inline test_journal.py:85]: out of curiosity, which of these two lines (84 & 85) actually raises the type error?

File: test_journal.py (hunk 2 above)
Comment (ndraper2) [inline test_journal.py:15]: The step 2 instruction manual specifically asks for the import at that line.  Which is correct?

File: test_journal.py (hunk 4 above)
Comment (ndraper2) [inline test_journal.py:85]: line 84 - it gets listed as a bad method call.  I'm not sure how exactly to handle the error in my code.

**Pull request merged**